### ISSUE TITLE
Konstig namngivning på variabel

### ISSUE THREAD

**Issue by glajan** [KasjanK/Modul-3-fetch-animals#1]
Variabeln borde heta "image"? getImage insinuerar att det skulle vara en funktion.
https://github.com/KasjanK/Modul-3-fetch-animals/blob/0af22a8fd8e6453b510bc271c244b78e265caafc/script.js#L2

**Comment by KasjanK** [comment]
Ja, det är sant. Glömde ändra namnet efter att jag testade en prototyp.

**Comment by KasjanK** [comment]
Uppdaterade namnet!
